### PR TITLE
deploy ktest

### PR DESCRIPTION
**Pull request by sean797**
I've made a few changes to the original `ktest` found https://theforeman.org/plugins/katello/developers.html

This allow you to run it from any directory & using relative paths to test files. e.g
```
[vagrant@centos7-devel test]$ pwd
/home/vagrant/katello/test
[vagrant@centos7-devel test]$ ktest models/content_view_test.rb controllers/api/v2/package_groups_controller_test.rb
``` 

OR

```
[vagrant@centos7-devel models]$ pwd
/home/vagrant/katello/test/models
[vagrant@centos7-devel models]$ ktest content_view_test.rb  -ntest_create
```

### CODE REVIEW

File: templates/ktest.sh.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+#
+# Deplyed with katello_devel scenario
+
+FOREMAN_PATH=<%= scope['::katello_devel::deployment_dir'] %>/foreman
+KATELLO_PATH=<%= scope['::katello_devel::deployment_dir'] %>/katello
+
+
+if [[ -n $1 ]]
```
Comment (ekohl) [inline templates/ktest.sh.erb]: This is a bashism. That's fine, but then the shebang should be ```#!/bin/bash```

Comment (sean797) [inline templates/ktest.sh.erb]: Thanks @ekohl I've updated it 👍

File: templates/ktest.sh.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  cd $FOREMAN_PATH
+  RAKE_PATH=`bundle show rake`
+  ruby -I"lib:test:${KATELLO_PATH}/test:${KATELLO_PATH}/spec" -I"${RAKE_PATH}/lib" $TEST_FILES $OTHER_OPTS
+else
+  cd $FOREMAN_PATH
+  bundle exec rake test:katello
+fi
+
+cd - > /dev/null
```
Comment (ekohl) [inline templates/ktest.sh.erb]: I wonder if this is actually needed. A script has its own working directory, right?

Comment (sean797) [inline templates/ktest.sh.erb]: Yes - you are right, its not required. Updated!

**Pull request merged**